### PR TITLE
Add CVX/USD and USD/CVX price identifier in DVM

### PR DESCRIPTION
**Pull request by petrovska-petro**
This UMIP is intended on tackling the addition of CVX/USD and USD/CVX to be queryable on-chain and to go thru UMA governance procedure

### CODE REVIEW

**Comment by smb2796** [comment]
Can you rename this file to `umip-cvx.md` or some equivalent?

**Comment by petrovska-petro** [comment]
> umip-cvx.md

Done, with the suggested name

File: UMIPs/umip-cvx.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+## Headers
+
+| UMIP-CVX            |                                                               |
+| ------------------- | ------------------------------------------------------------- |
+| UMIP Title          | Add CVX/USD and USD/CVX as supported price identifiers |
```
Comment (smb2796) [inline UMIPs/umip-cvx.md:5]: Price id naming convention suggests removing forward slashes. Can you change all of these to CVXUSD and USDCVX?

File: UMIPs/umip-cvx.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
+## Headers
+
+| UMIP-CVX            |                                                               |
+| ------------------- | ------------------------------------------------------------- |
```
Comment (smb2796) [inline UMIPs/umip-cvx.md:4]: Also would it be useful for your use case to also have CVXETH and ETHCVX price identifiers? Just seems low lift to add which is why I ask.

File: UMIPs/umip-cvx.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+- Market and pairs:
+    - CVX/USDT: [Okex](https://www.okex.com/markets/spot-info/cvx-usdt)
+    - CVX/WETH: [Sushiswap](https://analytics.sushi.com/pairs/0x05767d9ef41dc40689678ffca0608878fb3de906)
+    - CVX/WETH: [Uniswap-v3](https://info.uniswap.org/#/pools/0x2e4784446a0a06df3d1a040b03e1680ee266c35a)
+    - ETH/USD(T): Refer to ETHUSD in [UMIP-6](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-6.md)
+
+- Example data providers:
+    - CVX/USDT: CryptoWatch
+    - CVX/USD: Coingecko (ID: `convex-finance`)
```
Comment (smb2796) [inline UMIPs/umip-cvx.md:43]: Suggest removing coingecko as you don't actually use it in your price config. I'd just leave this as CWatch

File: UMIPs/umip-cvx.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+        uniswapAddress: "0x05767d9EF41dC40689678fFca0608878fb3dE906",
+        twapLength: 300
+      },
+      CVX_WETH_UNI: {
+        type: "uniswap",
+        version: "v3",
+        uniswapAddress: "0x2E4784446A0a06dF3D1A040b03e1680Ee266c35a",
+        twapLength: 300
+      },
+      CVX_USD_OKEX: { type: "cryptowatch", exchange: "okex", pair: "cvxusdt" },
```
Comment (smb2796) [inline UMIPs/umip-cvx.md:74]: You should be able to use the same 5-minute TWAP for Okex as you did for your dex source. Would follow the same pfconfig format of just specifying `twapLength: 300`.

File: UMIPs/umip-cvx.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+    },
+  },
+  "USD/CVX": {
+    type: "expression",
+    expression: "1 / CVX\\/USD",
```
Comment (smb2796) [inline UMIPs/umip-cvx.md:79]: If following my suggestion to change the price id names, you could also remove these escape characters.

File: UMIPs/umip-cvx.md (hunk 1 above)
Comment (petrovska-petro) [reply UMIPs/umip-cvx.md]: Follow the suggestion, changed in latest commit

File: UMIPs/umip-cvx.md (hunk 2 above)
Comment (petrovska-petro) [reply UMIPs/umip-cvx.md:4]: At the moment, for our potential use case, the CVXUSD and USDCVX will be sufficient

File: UMIPs/umip-cvx.md (hunk 3 above)
Comment (petrovska-petro) [reply UMIPs/umip-cvx.md]: Done in the last commit

File: UMIPs/umip-cvx.md (hunk 4 above)
Comment (petrovska-petro) [reply UMIPs/umip-cvx.md]: Included in the config expression in latest commit

File: UMIPs/umip-cvx.md (hunk 5 above)
Comment (petrovska-petro) [reply UMIPs/umip-cvx.md]: Remove also to you be "1/ CVXUSD" for this expression

File: UMIPs/umip-cvx.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+    - CryptoWatch: updated every trade
+    - AMM pools: updated every block mined
+
+# Price feed implementation
+
+This price identifier uses the [CryptoWatchPriceFeed](https://github.com/UMAprotocol/protocol/blob/master/packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.ts) and [UniswapPriceFeed](https://github.com/UMAprotocol/protocol/blob/master/packages/financial-templates-lib/src/price-feed/UniswapPriceFeed.ts) with the example configuration below:
+
+```
+  "CVXUSD": {
+    type: "expression",
```
Comment (Reinis-FRP) [inline UMIPs/umip-cvx.md:54]:
```suggestion
    type: "expression",
    expression: `
```

`expression` keyword was missing

Comment (petrovska-petro) [reply UMIPs/umip-cvx.md:54]: @Reinis-FRP thanks for pointing out, included in last commit

**Review by Reinis-FRP — APPROVED** [review]

File: UMIPs/umip-cvx.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+
+CVX token has predominant volume in Sushiswap at the time of writing the UMIP with average volume of 11M, folowed by the Uniswap pool with average volume of 700k over the past month, which will be supplemented by the CEXs feed. For this price identifier it is also assumed that 1 USDT = 1 USD holds at all times.
+
+CVX has predominant liquidity and volume activity in the AMMs paired with WETH. In order to mitigate attempted price manipulation 5 minute TWAP would be applied.
+
+
+# Implementation
+
+```
+1. Query CVX/WETH price from Uniswap v3 and SushiSwap using 5-minute TWAP.
```
Comment (therealhexi) [inline UMIPs/umip-cvx.md:107]: Any reason behind 5 minute TWAP? Usually it uses 15 minutes TWAP

**Review by therealhexi — APPROVED** [review]
Looks good to me

**Pull request merged**